docs/USERGUIDE.md

```diff
@@ -57,3 +57,188 @@ $ make clean
 # Clean only CRD
 $ make uninstall
 ```
+
+## Use external ceph cluster in Kubernetes
+This is a guide for connecting external ceph cluster to k8s using [Ceph CSI](https://github.com/ceph/ceph-csi). Ceph CSI plugins implement an interface between Kubernetes and Ceph cluster.
+Independent CSI plugins are provided to support RBD and CephFS backed volumes. The Ceph CSI version used in this guide is [v3.1.2](https://github.com/ceph/ceph-csi/releases/tag/v3.1.2).
+
+### 1. Get information of ceph cluster
+#### Install ceph cluster separately
+```shell script
+# On the node where the ceph cluster is installed
+$ cd /etc/ceph
+```
+
+##### 1-1. Ceph cluster information
+```shell script
+$ cat ceph.conf
+[global]
+	fsid = 50611be6-33b3-11eb-a5cb-0894ef32cba4
+	mon_host = v2:172.21.3.8:3300/0,v1:172.21.3.8:6789/0
+```
+
+##### 1-2. Key of the admin client
+```shell script
+$ cat ceph.client.admin.keyring
+[client.admin]
+	key = AQCeBcZftAEvExAAultsKBpNpiWWGi06Md7mmw==
+```
+
+##### 1-3. Name of the pool to be used for rbd
+```shell script
+$ ceph osd lspools
+1 device_health_metrics
+2 test-pool
+3 rbd-pool
+```
+
+##### 1-4. Name of the volume to be used for cephfs
+```shell script
+$ ceph fs volume ls
+[
+    {
+        "name": "myfs"
+    }
+]
+```
+
+#### Install ceph cluster using HyperSDS-Operator
+> Note: This part will be added soon.
+
+### 2. Deployment with K8s
+#### Deploy namespace
+```shell script
+# Namespace where the csi plugin pods and other resources are created
+$ kubectl apply -f deploy/csi/namespace.yaml
+```
+#### Deploy config
+Modify the value of `clusterID` and `monitors` in `deploy/csi/csi-config-map.yaml`.
```

Comment:
Could we explain more detail with yaml example?

Reply:
I'll add a comment to the values that need to be modified.
`# UPDATE THIS VALUE`
`adminKey: AQCeBcZftAEvExAAultsKBpNpiWWGi06Md7mmw==`